util/pagination/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Mind explaining me what is this `package.json` here?

Reply:
This was an "overlooked" file after deciding to ditch out a supposedly another pagination class (non-embed, which was the file pagination.js). Same with /pagination.js below. Will remove them on the next push.